browser.js

```diff
@@ -1,4 +1,5 @@
 var createHmac = require('create-hmac')
+var MAX_INT = Math.pow(2, 30) - 1 // default in iojs
```

Comment:
@calvinmetcalf renamed it to `MAX_ALLOC` in https://github.com/crypto-browserify/pbkdf2/commit/9f6fb05b7284bc0fc72cde6e37524deffdc2d3a8